agent/acs/model/ecsacs/api.go

```diff
@@ -413,7 +413,7 @@ func (s ECRAuthData) GoString() string {
 	return s.String()
 }
 
-type EFSVolumeConfiguration struct {
+type EfsVolumeConfiguration struct {
```

Comment:
Did you use gogenerate to generate the api.go file? The struct name should still be**EFSVolumeConfiguration**.